.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Test
+
+on: [push, pull_request]
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
+        os: [ubuntu-latest, ubuntu-20.04]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Get pip cache dir
+        id: pip-cache
+        run: |
+          echo "::set-output name=dir::$(pip cache dir)"
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.pip-cache.outputs.dir }}
+          key:
+            ${{ matrix.os }}-${{ matrix.python-version }}-${{ hashFiles('**/setup.py') }}
+          restore-keys: |
+            ${{ matrix.os }}-${{ matrix.python-version }}-
+
+      - name: Install dependencies
+        shell: bash
+        run: |
+          make install install-docs
+
+      - name: Run Tests
+        shell: bash
+        run: |
+          (source venv/bin/activate && make black check_conftest_imports doctest coverage doc-build)
```

Comment:
nit: I don't think the source is needed anymore -- it's addressed in the Makefile as of 2ae329b37f5960b2cb5aa7e677e0e81ef7527bca

Reply:
dropped!

Reply:
hrm failing https://github.com/mozilla/frost/pull/432/checks?check_run_id=1670520524

`make install` makes the venv but the other targets fail in `check_venv`